### ISSUE TITLE
[image_picker] No response when picking a PIC image on iOS 

### ISSUE THREAD

**Issue by ZhengChunLin** [flutter/flutter#126625]
### Is there an existing issue for this?

- [X] I have searched the [existing issues](https://github.com/flutter/flutter/issues)
- [X] I have read the [guide to filing a bug](https://flutter.dev/docs/resources/bug-reports)

### Steps to reproduce

I Use 
image_picker 0.8.7+4
flutterSDK 3.7.12
dart 3.64.0
iPadOS 16.4.1

pick *.PIC file from gallery

### Expected results

Future (final image = await _picker.pickImage(source: ImageSource.gallery)) returns image data.

### Actual results

No result returned, future hangs and no error comes.

### Code sample

I Use image_picker [sample code](https://github.com/flutter/packages/tree/main/packages/image_picker/image_picker/example)

### Flutter Doctor output

wanlixin@wanlixindeAir mobileApp % flutter doctor -v
[✓] Flutter (Channel stable, 3.7.12, on macOS 13.3.1 22E261 darwin-x64, locale zh-Hans-CN)
    • Flutter version 3.7.12 on channel stable at /Users/wanlixin/flutter/flutter_SDK
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 4d9e56e694 (3 weeks ago), 2023-04-17 21:47:46 -0400
    • Engine revision 1a65d409c7
    • Dart version 2.19.6
    • DevTools version 2.20.1
    • Pub download mirror https://pub.flutter-io.cn
    • Flutter download mirror https://storage.flutter-io.cn

[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
    • Android SDK at /Users/wanlixin/Library/Android/sdk
    • Platform android-33, build-tools 33.0.0
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 14.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 14E222b
    • CocoaPods version 1.11.3

[✓] Android Studio (version 4.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)

[✓] VS Code (version 1.78.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.64.0

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!

**Comment by huycozy** [comment]
@ZhengChunLin Can you please share the image source file here?

**Comment by ZhengChunLin** [comment]
@huycozy  Sorry,PIC Images cannot be uploaded here.

**Comment by huycozy** [comment]
You can compress it and retry to upload

**Comment by ZhengChunLin** [comment]
[dress.PIC.zip](https://github.com/flutter/flutter/files/11483870/dress.PIC.zip)
@huycozy I understand. I have already compressed and uploaded it

**Comment by huycozy** [comment]
@ZhengChunLin Thanks. Which device are you checking this? 

I checked this on physical device (iPhone 7, iOS 15.7.2) and see it can work well. But on iOS emulator, the image is not even previewed on Photos native app, it only be listed on Files app but can't preview it:

<img width="400" src="https://github.com/flutter/flutter/assets/104349824/c4252da3-eddc-40ff-8317-7bcf0133744c">

**Comment by ZhengChunLin** [comment]
@huycozy  My iPad and iPhone 13 Pro Max are in the latest system (iOS 16.4.1), the album can be previewed, but there is no response to upload. When I debug, the selected image does not respond. `final image = await _picker.pickImage(source: ImageSource.gallery)`

**Comment by huycozy** [comment]
Are you testing this on physical device or emulator? It seems it only can be previewed on a physical device. But I don't have such device (model and iOS version). 

Also, it's worth giving it a try on another iOS native application to see if that image can be previewed and picked or not. I'm wondering this because it doesn't seem to be supported by the plugin (see [this](https://github.com/flutter/packages/blob/329f83c7b9898f774a2abd49b31a5a7c261cd5f5/packages/image_picker/image_picker_ios/ios/Classes/FLTPHPickerSaveImageToPathOperation.m#L92-L94))

**Comment by ZhengChunLin** [comment]
@huycozy I used several of my friends' phones and chose the. PIC image format, but there was no response.

Your device(iPhone 7, iOS 15.7.2)  is working properly and can view image information in the device album. PIC format will not be displayed. As shown in the figure

![IMG_0328](https://github.com/flutter/flutter/assets/22043508/6c2fdc35-6aca-47e7-b31f-f54d113f7412)
![IMG_0329](https://github.com/flutter/flutter/assets/22043508/ef47537c-5684-45ef-99b9-01f5c378b742)

Do you have any plans to support this image format?

**Comment by huycozy** [comment]
The image is displayed as PNG on my device (this can explain why it works as expected). 
Can you please confirm:
- Are those screenshots above on two different devices? 
- Which of them can work with `image_picker`?

>Do you have any plans to support this image format?

I'm not sure if this is an issue or proposal for this format until verify this on other native iOS apps as I mentioned above [comment](https://github.com/flutter/flutter/issues/126625#issuecomment-1550626956). It could also be an OS or image issue I think. So I would recommend trying the same function with that image on another native app (which has picking image function) to see if it can work normally.

**Comment by ZhengChunLin** [comment]
> Are those screenshots above on two different devices?
No, On the same iPad device.  Image without display format is `PIC`,  `JPEG` is a converted format image

>  Which of them can work with `image_picker`?
 `JPEG` can be used.

`PIC` pictures can be displayed in albums and can be selected by `image_picker`. There's just no response after the selection.
Other apps on my phone can choose to use this image normally.

**Comment by huycozy** [comment]
Labeling the issue for further insights. 

### Summary
- I can't reproduce this issue on my device (iPhone 7, iOS 15.7.2) as it's not previewed image. 
- @ZhengChunLin's [devices](https://github.com/flutter/flutter/issues/126625#issuecomment-1549069751) can preview the PIC image but `image_picker` plugin can not respond after picking it.

**Comment by stuartmorgan** [comment]
This may be a duplicate of the other iOS image picker simulator-specific issues.

**Comment by NischalJoshi777** [comment]
In my case, the camera/gallery is opening pretty late after restarting the app. It's taking me almost 10-15 secs to open the camera in the initial case. 
Device: iPhone 14 pro max. 
iOS version: 16.4.1

**Comment by aniket-github** [comment]
I am trying to upload image
https://drive.google.com/file/d/17UVDvszLZb1Mlu3vUuRa6QPvlCev9_ee/view?usp=share_link
wide camera - 26 mm f1.5
RAW

**Comment by hellohuanlin** [comment]
@ZhengChunLin I've never seen this format (PIC) before. May I ask where it's created? Maybe try a dummy UIImagePicker native app and see if we can pick the image and display it in a UIImageView
